script.js

```diff
@@ -0,0 +1,110 @@
+let awesomeBooks = [];
+let bookIncrement = 0;
+function Book(bookTitle, bookAuthor, bookId) {
+  this.bookTitle = bookTitle;
+  this.bookAuthor = bookAuthor;
+  this.bookId = bookId;
+}
+
+function getBooks() {
+  // localStorage.clear();
```

Comment:
- [ ] Kindly remove all commented lines of code in your project: it makes your codebase looks messy and difficult to read. If you need to keep track of previous lines/blocks of code, please use git history.